services/job_runner_test.go

```diff
@@ -219,6 +218,40 @@ func TestJobRunner_ExecuteRun_ErrorsWithNoRuns(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
+func TestJobRunner_BeginRunWithAmount(t *testing.T) {
+	tests := []struct {
+		name   string
+		amount *big.Int
+		status models.RunStatus
+	}{
+		{"job with no amount", nil, models.RunStatusCompleted},
+		{"job with zero amount", big.NewInt(9), models.RunStatusErrored},
```

Comment:
Looks like this case is mislabelled, a combination of the zero case and the just under case.